bodhi/server/models.py

```diff
@@ -789,6 +789,14 @@ class Update(Base):
     user_id = Column(Integer, ForeignKey('users.id'))
 
     @property
+    def mandatory_days_in_testing(self):
+        if self.critpath:
+            return int(config.get('critpath.stable_after_days_without_negative_karma'))
+
+        days = self.release.mandatory_days_in_testing
+        return days if days else 0
```

Comment:
How can I achieve test number 3? There should be a release with `mandatory_days_in_testing` equals to 0. Is there a way to mock that nicely?

My current solution looks like [this](https://gist.github.com/nbianca/4e6a9c1a244395e46ca4de0251c73d0e).